src/main/java/entities/MonthObject.java

```diff
@@ -2,6 +2,6 @@
 
 /**
  * An interface that objects belonging to the MonthlyStorage implement. Expense and Category implement this so that
- * they can be created by MonthObjectCreator classes.
+ * they can be created by MonthObjectFactory classes.
  */
 public interface MonthObject {}
```

Comment:
The interface does not contain any methods? Shouldn't it have methods like getName() setName() getValue() setValue() so they could be called in the use case interactor?

Reply:
Maybe it'll be some getters and setters for the mutual items such as name and value, that will be required for the classes that implement MonthObject (in our case Expense and Category). This in turn can be used in the use case interactor of the Add/Edit features!

Reply:
Hmm that was something I thought about, but since value is budget in Category, I decided against it. I think i will add name to both, since we can agree those are the same in both Category and Expense, and every MonthObject should have a name!